app/bin/run-ecs-task/README.md

```diff
@@ -0,0 +1,62 @@
+# Run Task
+The `run-task.sh` is a wrapper shell script around the AWS run task command for starting ECS tasks. It helps setup and find the parameters necessary for starting an ECS task.
+
+# Setup
+This command requires you to have the [AWS CLI](https://aws.amazon.com/cli/) setup and configured locally.
+
+It also heavily uses `jq` for parsing and generating the CSV configurations.
+
+# Usage
+
+`./run-task.sh -a your.name -e test -t your-task-definition` is the most basic way to run a task.
+
+The script has the following parameters:
+* `-a` - The author running the script (eg. if your name is Bob Smith, put `-a bob.smith`)
+* `-e` - The environment you want to run the script in (eg. `-e test`)
+* `-t` - The ECS task definition you want to run (Your task definitions can be found at https://us-east-1.console.aws.amazon.com/ecs/home#/taskDefinitions , eg `-t db-migrate-up`)
+* `-s` - (Optional) - The subnets the ECS task could run in (eg. `-s '["subnet-123456789", "sg-567890654"]'`)
+    * If no subnet is specified, the script will fetch all subnets attached to the account.
+* `-g` - (Optional) - The security group IDs to attach to the ECS task (eg. `-g '["sg-1234", "sg-3456"]'`)
+    * If no security groups are specified, the `csv-handler` security group is fetched (TODO)
+
+## Template Files
+Two JSON template files exist which are used to help format the JSON configurations.
+
+### Network config
+The network config should be left as-is, as the subnet and security groups are specified as described in the usage section above.
+
+### Container overrides
```

Comment:
Could you add some more description here of how one should use the container override json? Is the idea that I would edit the file if I wanted to change the default override?

Reply:
Yes, that's right. I can add more detail.